<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HustMap</class>
 <widget class="QMainWindow" name="HustMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HustMap</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="gfxview">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>700</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>20</y>
      <width>241</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vlayout">
     <item>
      <layout class="QHBoxLayout" name="hlayout1">
       <item>
        <widget class="QLabel" name="srcLabel">
         <property name="text">
          <string>起点(左键选择)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="srcPos">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlayout2">
       <item>
        <widget class="QLabel" name="dstLabel">
         <property name="text">
          <string>终点(右键选择)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dstPos">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlayout3">
       <item>
        <widget class="QLabel" name="numLabel">
         <property name="text">
          <string>线路数目</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="numValue">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlayout4">
       <item>
        <widget class="QLabel" name="transLabel">
         <property name="text">
          <string>转乘次数</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="transValue">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlayout5">
       <item>
        <widget class="QLabel" name="searchLabel">
         <property name="text">
          <string>搜索地点</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchEdit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="searchResult">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
